<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayScene</class>
 <widget class="QMainWindow" name="PlayScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>选择你要玩的谱面（￣︶￣）↗　</string>
  </property>
  <property name="windowIcon">
   <iconset resource="testres/testres.qrc">
    <normaloff>:/test/nku.png</normaloff>:/test/nku.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Backbtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="MusicSelectWindow" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>269</width>
       <height>159</height>
      </rect>
     </property>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>test</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>the story unfolds</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>paradise</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Playbtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string>label</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>Backbtn</zorder>
   <zorder>scrollArea</zorder>
   <zorder>Playbtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicSelectWindow</class>
   <extends>QScrollArea</extends>
   <header>musicselectwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="testres/testres.qrc"/>
 </resources>
 <connections/>
</ui>
